.github/workflows/python-app.yml

```diff
@@ -49,14 +49,14 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flake8 pytest black isort codecov pytest-cov
+          pip install flake8==3.8.3 pytest black==20.8b1 isort==5.4.2 codecov pytest-cov
```

Comment:
Why should we fix versions though? Isn't it counter intuitive? Whenever there is an upgrade we'd need to change this.

Reply:
I think the issue was created because was we havent updated our versions locally, maybe we should do that. I'm not 100% sure about the pros and cons of the two paths.

Reply:
The main problem is that we don't know when isort and black become incompatible with each other. We'll have to sort every error as it comes. Besides isort and black just help us adhere to the same coding norms. It doesn't matter what those norms are.

Reply:
@Sharad24 is that fine?

Reply:
Yeah sounds good.